internal/pkg/aws/apprunner/apprunner_test.go

```diff
@@ -77,6 +80,12 @@ func TestAppRunner_DescribeService(t *testing.T) {
 						Value: "info",
 					},
 				},
+				EnvironmentSecrets: []*EnvironmentSecret{
```

Comment:
nit: can we add another value here to test the sorting? for example `parameter/zzz123` to make sure it appears after `jan11ssm`

Reply:
I added that, and checked. Test case Passed!